src/main/java/com/griefcraft/modules/flag/MagnetModule.java

```diff
@@ -141,36 +144,75 @@ public void run() {
                             continue;
                         }
 
-                        Location location = item.getLocation();
-                        int x = location.getBlockX();
-                        int y = location.getBlockY();
-                        int z = location.getBlockZ();
-
-                        List<Protection> protections = lwc.getPhysicalDatabase().loadProtections(world.getName(), x, y,
-                                z, radius);
-                        for (Protection protection : protections) {
-                            if (protection.hasFlag(Flag.Type.MAGNET)) {
-
-                                if (protection.getBukkitWorld().getName() != item.getWorld().getName())
```

Comment:
Oh it also fixes the issue here where strings are compared with `==`, regardless of this PR I recommend pulling this in.